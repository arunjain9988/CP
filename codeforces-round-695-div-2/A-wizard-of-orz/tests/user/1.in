3
52
15
256