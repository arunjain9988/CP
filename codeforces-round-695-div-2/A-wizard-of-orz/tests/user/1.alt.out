9890123456789012345678901234567890123456789012345678
989012345678901
9890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012
